governance.md

```diff
@@ -0,0 +1,145 @@
+# Governance
+
+[mdn-browser-compat-data](https://github.com/mdn/browser-compat-data) (also often referred to as "BCD") is an open source project that depends on contributions from the community. Anyone may contribute to the project at any time by submitting code, participating in discussions, making suggestions, or any other contribution they see fit. This document describes how various types of contributors work within the mdn-browser-compat-data project.
+
+## Roles and Responsibilities
+
+### Everyone
+Everyone who is involved in any form with the project must abide by the project’s [Contribution Guidelines](https://github.com/mdn/browser-compat-data/blob/master/CODE_OF_CONDUCT.md). Everyone is expected to be respectful of fellow community members and to work collaboratively in the spirit of inclusion.
+
+### Users
+Users are community members who have a need for the project. They are typically consumers of the compat data (see [data consumers](https://github.com/mdn/browser-compat-data#projects-using-the-data)). Anyone can be a User; there are no special requirements and the data is licensed under [CC0](https://github.com/mdn/browser-compat-data/blob/master/LICENSE). Common User contributions include evangelizing the project (e.g., display a link on a website and raise awareness through word-of-mouth), informing developers of strengths and weaknesses from a new user perspective, or providing moral support (a “thank you” goes a long way).
+
+Users who continue to engage with the project and its community will often become more and more involved. Such Users may find themselves becoming [Contributors](#Contributors), as described in the next section.
+
+### Contributors
+Contributors are community members who contribute in concrete ways to the project, most often in the form of data updates, code and/or documentation. Anyone can become a Contributor, and contributions can take many forms. There is no expectation of commitment to the project, no specific skill requirements, and no selection process.
+
+Contributors:
+- Have read-only access to source code and therefore can submit changes via pull requests.
+- Have their contribution reviewed and merged by a [Peer](#Peers) or [Owner](#Owners). Owners and Peers work with Contributors to review their code and prepare it for merging.
+- May also review pull requests. This can be helpful, but their approval or disapproval is not decisive for merging or not merging PRs.
+
+As Contributors gain experience and familiarity with the project, their profile within, and commitment to, the community will increase. At some stage, they may find themselves being nominated for becoming a Peer by an existing Peer or Owner.
+
+### Peers
+Peers are community members who have shown that they are committed to the continued development of the project through ongoing engagement with the community. Peers are given push/write access to the project’s GitHub repos.
```

Comment:
Is it an _expectation_ that peers should review and merge data PRs? I mean this doc says that they _can_, but is signing up to some of that workload an expectation?

Reply:
This is a valid question. I help out whenever I can, but MDN doesn't pay my salary.

Reply:
I think being a peer brings wiith it some expectation that you will do some data reviewing/merging, otherwise there is not much point you having the access privs you are given. If you don't do many PR reviews/merges, I guess it would still be acceptable peer duties to do other things like contribute to strategy dicussions and help with updates to the underlying code/other structures.

It is something we'll have to use our judgement on — if someone is contributing constructively, then fine.

Joe, I think the amount of input you've made definitely makes you elligable for consideration as a peer. I think many/most of the contributors to the project will end up being volunteers, not paid by MDN, and we will be mindful of that.

Reply:
As Chris says: There is an expectation that peers do something with their peer status because otherwise there is no need for them to have these access privileges. However, there is no expectation like "must review 10 PRs a week" or similar. The hope is that we will share workload more among the peers, but no one will require from anyone to get a specific amount of work done.